plugins/source/aws/client/services.go

```diff
@@ -255,8 +257,8 @@ type Services struct {
 	Xray                         services.XrayClient
 }
 
-func (s *Services) InitService(awsConfig *aws.Config, service AWSServiceName) {
-	c := awsConfig.Copy()
+func (s *Services) InitService(service AWSServiceName) {
+	c := s.config
```

Comment:
To be on the safe side, can we keep the `Copy()` in here for now? I don't know if any of these `NewFromConfig` functions might actually mutate it.

Reply:
2b5d1ac